spec/helpers/barnardos/action_view/form_builder_spec.rb

```diff
@@ -67,9 +67,10 @@ class TestHelperCarrier < ActionView::Base
 
     let(:collection) do
       {
-        team:     'Just the team',
-        internal: 'Other teams internally',
-        external: 'Other teams externally'
+        anonymised: 'The first label',
+        team:       'The second label',
+        internal:   'The third label',
+        external:   'The forth label'
```

Comment:
Slight typo, should read 'the fourth label'. Affects nothing. Except for me writing this comment.